src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -220,5 +240,34 @@ public static void ValidateSslCertificate(HttpClient client)
                 );
             }
         }
+
+        /// <summary>
+        /// Format request information when LogLevel is set to debug
+        /// </summary>
+        ///
+        /// <param name="request">HTTP request</param>
+        private static void logRequest(Request request){
+            Console.WriteLine("-- BEGIN Twilio API Request --");
+            Console.Write(request.Method + " ");
```

Comment:
Why a trailing space?

Reply:
The method and URI were originally logged on the same line with no delineation between them but I decided to move them to two separate labeled lines.